core/domain/feedback_services_test.py

```diff
@@ -400,6 +401,33 @@ def test_get_thread_summaries(self):
             # Check if the summaries match.
             self.assertEqual(thread_summary, thread_summaries[index])
 
+    def test_get_thread_summaries_load_test(self):
+        # The speed of fetching the summaries of 100 threads having 5 messages
+        # should be less than 8 seconds.
```

Comment:
Also, you don't want to be "brushing up" against the 1 second limit. It should be comfortably under. Otherwise what you're gonna end up with is flaky backend tests.